stubs/docutils/docutils/io.pyi

```diff
@@ -23,6 +23,18 @@ def error_string(err: BaseException) -> str: ...
 class Input(TransformSpec):
     component_type: ClassVar[str]
     default_source_path: ClassVar[str | None]
+    def __init__(
+        self,
+        source: str | None = None,
+        source_path: str | None = None,
+        encoding: str | None = None,
+        error_handler: str = "strict",
+    ) -> None: ...
+    encoding: str | None
+    error_handler: str
+    source: str | None
+    source_path: str | None
+    successful_encoding: str | None = None
```

Comment:
Nit: Could you move this above the `__init__` method as is customary in typeshed?